### ISSUE TITLE
Black Screen

### ISSUE THREAD

**Issue by OBS1227** [royshil/obs-backgroundremoval#59]
Hi, 
The most recent updates by background remover and OBS has stopped the crashing, but I choose facetime camera then background remover then all the BR options are their before I turn on the BR, but when I turn on BR I see a black screen no facetime of me , but have the option to select a new color. 

Mac OS High Sierra 
OBS 27.1.3. 
Background removal - Current

This is from the log file.

https://obsproject.com/logs/C-IaBJTW7dqIzClK

Thanks so much!

**Comment by royshil** [comment]
@OBS1227  the problem is 
```
01:58:02.112: [obs-backgroundremoval] Unable to get model filename mediapipe.onnx from plugin.
```
It unable to find the model files that are packaged with the plugin
How are you installing the plugin? are you following the instructions?
Ideally I'd need to take a peek at the location where the plugin is installed to see where the files are.

**Comment by OBS1227** [comment]
Yes, that maybe the problem? I drag your OBS-Background removal.so  into the plug in section of the OBS  plugins. I had watched a video many months ago on how to install and may have forgotten some additional installations. Did I? It shows up like the other ones, but black screen. Thank You so much for your time and effort in trying  to help me figure this out. I can see the possibilities.

**Comment by royshil** [comment]
@OBS1227  dragging just the .so file is not enough... you need the model files.
You can follow the install instructions on https://github.com/royshil/obs-backgroundremoval/releases/tag/v0.4.0 to install
Basically it's something like `$ unzip -o obs-backgroundremoval-macosx.zip -d /Applications/OBS.app/Contents/` once you download the .zip file

**Comment by OBS1227** [comment]
I am really sorry. I don't know how to do the next step. I tried to google an installation like yours, but couldn't find one. Is it enough to take the remaining file and drag/dop it into the OBS folder to fix it? I see programming argon and I don't know where it goes. Again, I wish I understood or have some experience with this, but I don't. Sorry for the inconvenience!

**Comment by OBS1227** [comment]
I'm sorry I don't understand some of the programming language. $ unzip -o I don't understand. Unless it simply means to just unzip the file. After reading it again does the whole instruction mean to just drag the whole folder after it is unzipped into the contents folder of OBS? I'll give it a try until you get back to me.....if not, I'll try to take a snapshot and you can tell me what I'm doing wrong. Thanks so much!

**Comment by OBS1227** [comment]
Ok. So at first I assumed the model files are included as part of the initial zip files, but now it appears I need to look for a nd download file? Or, it is included in the initial download and I need to know where to put it? Not sure what  snapshot will do. I just need to put the appropriate files in the appropriate folder. Thanks.

**Comment by thatcomicsguy** [comment]
The install instructions on Github are always kind of cryptic and unsettling for dummies like me.  They rarely match anything I actually see on my computer screen and I have to play "guess-what-the-engineer-means".  Or I'm asked to use commands like, "dollar sign something something" which means.., I have no idea what.  I'm using Winzip with drag and drop features and some dropdown menus.

Sometimes I can figure this stuff out, but this time it's not working. I honestly can't tell if I'm installing things correctly.

On my Windows 10 x64 install of the latest OBS, the directory structure and that of the download don't match those indicated by the instructions...

On my system, this is what I find...

**Program Files > obs-studio**
Which contains the following folders...
**"bin" "data" and "obs-plugins"**

And the unzipped file downloaded from Github offers a folder named...
**"obs-backgroundremoval-0.4.0-win64"**

I dropped that in with the "bin" "data" and "obs-plugins" folders and figured, "That's totally not going to work" so I also put a copy in the "obs-plugins" folder, which seemed intuitive enough.

I ended up scattershot putting a bunch of copies in different variations throughout OBS's file structure in places which seemed to be appropriate, (like putting the three .dll files found after drilling down the file structure of the download) into:

Program Files > obs-studio > obs-plugins > 64bit
(where a bunch of other .dll files live).

Finally!  Progress.  Now the filter appears on the list once OBS is started up and it can be selected, but all I get is a black screen no matter how I fiddle with the sliders.

If the devs here have the time, it would be seriously cool if they could write a "For Dummies" version of the install instructions.

Thank-you!
